enterprise/ha/src/test/java/org/neo4j/kernel/ha/TxPushStrategyConfigIT.java

```diff
@@ -71,17 +77,27 @@ public void twoRoundRobin() throws Exception
     {
         startCluster( 4, 2, "round_robin" );
 
-        createTransactionOnMaster();
-        assertLastTransactions( lastTx( FIRST_SLAVE, BASE_TX_ID + 1 ), lastTx( SECOND_SLAVE, BASE_TX_ID + 1 ),
-                lastTx( THIRD_SLAVE, BASE_TX_ID ) );
+        long txId = getLastTx( cluster.getMaster() );
+        int count = 15;
+        for ( int i = 0; i < count; i++ )
+        {
+            createTransactionOnMaster();
+        }
 
-        createTransactionOnMaster();
-        assertLastTransactions( lastTx( FIRST_SLAVE, BASE_TX_ID + 1 ), lastTx( SECOND_SLAVE, BASE_TX_ID + 2 ),
-                lastTx( THIRD_SLAVE, BASE_TX_ID + 2 ) );
+        long min = -1, max = -1;
+        for ( GraphDatabaseAPI db : cluster.getAllMembers() )
```

Comment:
Shouldn't we exclude the master here? Otherwise the assertions are trivially correct since the master is always up-to-date.